src/styles/common/_login.scss

```diff
@@ -0,0 +1,13 @@
+.login__container {
+  background: url(https://i.ibb.co/Bfd2d2B/Hero-bg.png);
```

Comment:
Same as what @gsluthra mentioned. This is an external image which we should not be using (even if it is a stock/demo image)

Reply:
Okay, so no images should be included?

Reply:
Check with @gsluthra if we have an image reference set available or how we go about handling open license images